### ISSUE TITLE
[KNX2] Rollershutters can not be set to a percent value

### ISSUE THREAD

**Issue by juelicher** [openhab/openhab2-addons#3415]
With the KNX 1.x binding I could set a rollershutter to half closed by issuing “sendCommand(Rollaeden_Kueche, 50)”. With the KNX 2.x binding this has no effect (I also tried the Rollaeden_Kueche.sendCommand(50)).

Moving the rollershutter by pressing the up or down arrows in the UI works as expected, so the configuration should be correct.

For reference the configuration for this rollershutter:

`Type rollershutter           : Rollaeden_Kueche   "Rollaeden Kueche"   [ upDown="4/2/100", stopMove="4/2/101", position="4/3/100" ]`

`Rollershutter Rollaeden_Kueche   "Rolläden"   (gKueche)   { channel="knx:device:bridge:knx-1_1_101:Rollaeden_Kueche" }`

**Comment by juelicher** [comment]
For reference here  my currently workink KNX 1.x configuration:

`Rollershutter Rollaeden_Kueche                      "Rolläden"                                      (gKueche)                                                                               { knx="4/2/100, 4/2/101, 4/3/100" }`

With this configuration setting the position works as expected, just tested it to be sure.  So 4/3/100 seems to be correct.

**Comment by juelicher** [comment]
Because of your "works for me" remark I did some further testing. And indeed, sending a percent value to the rollershutter item works e.g. when sending it from the karaf console.

What did not work was the "received command" trigger of the rule that sends the value to the rollershutter, so the send command was not executed in the first place. As this rule worked for years and works correctly with openhab 2.2 and KNX 1.x I am not sure, if this is a openHAB 2.3 or a KNX 2 binding issue.

To discuss this topic I posted in the community forum: [](https://community.openhab.org/t/knx-2-received-command-rule-trigger-does-not-work/42857)

Sorry for this false bug report!
